src/circuit/gadget/ecc/chip.rs

```diff
@@ -219,10 +207,6 @@ impl EccChip {
             meta.enable_equality((*column).into());
         }
 
-        let q_mul_fixed_running_sum = meta.selector();
-        let running_sum_config =
-            RunningSumConfig::configure(meta, q_mul_fixed_running_sum, advices[4]);
```

Comment:
Moving these into `mul_fixed::Config::configure` causes constraint reordering. If we do want to do this, we should do it in a separate commit, so that the `mul_fixed::Config` refactor does not include constraint reordering.